pkg/apis/config/defaults/image.go

```diff
@@ -18,4 +18,4 @@ limitations under the License.
 package defaults
 
 // Image is the default for the Config.Image field, aka the default node image.
-const Image = "kindest/node:v1.20.2@sha256:8f7ea6e7642c0da54f04a7ee10431549c0257315b3a634f6ef2fecaaedb19bab"
+const Image = "kindest/node:v1.20.2@sha256:15d3b5c4f521a84896ed1ead1b14e4774d02202d5c65ab68f30eeaf310a3b1a7"
```

Comment:
note: we are not rolling forward the default kubernetes version yet due to upstream startup time regression.

TODO: probably in 1.21 after other things settle down we'll start tracking this in CI.